tracer/src/Datadog.Trace/ClrProfiler/AutoInstrumentation/AdoNet/DbScopeFactory.cs

```diff
@@ -66,9 +66,10 @@ private static Scope CreateDbCommandScope(Tracer tracer, IDbCommand command, Int
                     IastModule.OnSqlQuery(commandText, integrationId);
                 }
 
-                if (tracer.Settings.DbmPropagationMode != DbmPropagationLevel.Disabled && (integrationId == IntegrationId.MySql || integrationId == IntegrationId.Npgsql))
+                if (tracer.Settings.DbmPropagationMode != DbmPropagationLevel.Disabled && (integrationId == IntegrationId.MySql || integrationId == IntegrationId.Npgsql || integrationId == IntegrationId.SqlClient))
```

Comment:
nit: might be easier to read with pattern matching
```suggestion
                if (tracer.Settings.DbmPropagationMode != DbmPropagationLevel.Disabled && (integrationId is IntegrationId.MySql or IntegrationId.Npgsql or IntegrationId.SqlClient))
```

Reply:
Nitpicking and not related to the PR. Clearly feel free to ignore. 

You could move all this logic in `DatabaseMonitoringPropagator` to have all the DBM stuff in the same class (that can help utesting some times for instance).

So you could call `DatabaseMonitoringPropagator.Propagate(command, integrationId, tracer.Settings)` and put the if, the computation of propagationMode there.

But again, feel free to ignore.

Reply:
I totally agree with you Pierre, for the first try I decided to make these changes and check after if it makes more sense moving that logic to the **DatabaseMonitoringPropagator** class, I thought of it more because of the multiple databases or changes we may need to end up doing for feature.